packages/wonder-blocks-dropdown/components/action-menu-opener-core.js

```diff
@@ -81,11 +81,11 @@ export default class ActionMenuOpenerCore extends React.Component<Props> {
                 aria-expanded={opened ? "true" : "false"}
                 aria-haspopup="menu"
                 aria-label={ariaLabel}
-                data-test-id={testId}
                 disabled={disabled}
                 style={defaultStyle}
                 type="button"
                 {...handlers}
+                data-test-id={testId}
```

Comment:
For some reason, data-test-id was being overwritten before.

Reply:
I thought maybe `testId` might have been included in `...handlers`, but that appears not to be the case.